### PR TITLE
Version mobile report page

### PR DESCRIPTION
**Pull request by Keikonichi**
Rewriting some properties to make the report page responsive

### CODE REVIEW

**Comment by Keikonichi** [comment]
The report page response is incomplete in screen sizes from 780px to 1200px

File: src/components/ClaimReview/SentenceReportCard.tsx [hunk 1]
```diff
@@ -5,111 +5,213 @@ import colors from "../../styles/colors";
...
+    context: any;
+}) => {
     const { t } = useTranslation();
     const speechTypeTranslation =
-        claimType === 'speech'
-            ? t('claim:typeSpeech')
-            : t('claim:typeTwitter')
+        claimType === "speech" ? t("claim:typeSpeech") : t("claim:typeTwitter");
+
+    const SentenceReportCardStyle = styled.div`
```
Comment (thesocialdev) [inline src/components/ClaimReview/SentenceReportCard.tsx:31]: Move to its own file as a new component.

File: src/components/ClaimReview/SentenceReportView.tsx [hunk 2]
```diff
@@ -9,141 +9,235 @@ import { LinkPreview } from "@dhaiwat10/react-link-preview";
...
         margin: "10px 0px",
         width: "100%",
         fontWeight: 400,
         fontSize: "18px",
-        lineHeight: "30px"
-    }
+        lineHeight: "30px",
+    };
+
+    const CardStyle1 = styled.div`
```
Comment (thesocialdev) [inline src/components/ClaimReview/SentenceReportView.tsx:36]: Use a more specific name and move to its own file.

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: src/components/ClaimReview/SentenceReportCard.tsx (hunk 1 above)
Comment (thesocialdev) [inline src/components/ClaimReview/SentenceReportCard.tsx:31]: Move to its own file.

File: src/components/ClaimReview/SentenceReportView.tsx [hunk 3]
```diff
@@ -9,141 +9,235 @@ import { LinkPreview } from "@dhaiwat10/react-link-preview";
...
+                                                <LinkPreview
+                                                    url={link}
+                                                    borderRadius="10px"
+                                                    backgroundColor={
+                                                        colors.lightGray
+                                                    }
+                                                    borderColor="transparent"
+                                                    imageHeight="156px"
+                                                    height={244}
+                                                    secondaryTextColor="#515151"
```
Comment (thesocialdev) [inline src/components/ClaimReview/SentenceReportView.tsx:183]: Use the colors library for every color added.

File: src/components/ClaimReview/NewCTARegistration.tsx [hunk 4]
```diff
@@ -7,43 +8,55 @@ function NewCTARegistration({ style = {} }) {
...
+                        onClick={() => {
+                            umami?.trackEvent(
+                                "cta-registration-button",
+                                "registration"
+                            );
+                        }}
+                        type={ButtonType.blue}
+                        target="_blank"
+                        href={t("common:registrationLink")}
+                        className="New-CTA-registration-button"
```
Comment (thesocialdev) [inline src/components/ClaimReview/NewCTARegistration.tsx:46]: For the CSS class names, we should use lower case everywhere.

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: src/components/ClaimReview/NewCTARegistration.tsx [hunk 5]
```diff
@@ -7,43 +8,55 @@ function NewCTARegistration({ style = {} }) {
...
+                        lineHeight: "21px",
+                        textAlign: "center",
                     }}
                 >
-                    {t("CTARegistration:button")}
-                </b>
-            </Button>
-        </div>
+                    {t(
+                        "AletheiaFact.org é um movimento e plataforma coletivo de verificação de fatos que imagina uma sociedade onde todos podem ter"
```
[the remark this answers is not in the document]
Comment (Keikonichi) [reply src/components/ClaimReview/NewCTARegistration.tsx:32]: This text in Figma is incomplete, how to proceed with the task?

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**